packages/dx-grid-core/src/plugins/table-column-resizing/helpers.ts

```diff
@@ -1,18 +1,64 @@
-import { ColumnSizeFn } from '../../types';
+import { ColumnSizeFn, ValidValueFn } from '../../types';
 
-export const getColumnSize: ColumnSizeFn = (
-  updatedColumn, {columnName, shift, minColumnWidth, maxColumnWidth, columnExtensions = [],
+export const getColumnsSizes: ColumnSizeFn = (
```

Comment:
Why this function names `getColumnsSizes` ? You are using the `s` word twice. Is it necessary?